garage/tf/models/gaussian_mlp_model.py

```diff
@@ -0,0 +1,194 @@
+"""Gaussian MLP Model."""
+import numpy as np
+import tensorflow as tf
+from tensorflow.keras.layers import Input
+from tensorflow.keras.layers import Lambda
+from tensorflow.keras.models import Model
+
+from garage.misc import ext
+from garage.tf.core.mlp2 import mlp2
+from garage.tf.core.parameterLayer import ParameterLayer
+from garage.tf.models import PickableModel
+
+
+class GaussianMLPModel(PickableModel):
+    """
+    GaussianMLPModel.
+
+    Args:
+        :param input_dim: Input dimension.
+        :param output_dim: Output dimension.
+        :param scope: Variable scope of the model.
+        :param hidden_sizes: List of sizes for the fully-connected hidden
+          layers.
+        :param learn_std: If std is trainable.
+        :param init_std: Initial std.
+        :param adaptive_std: If std is adaptive (modeled as a mlp).
+        :param std_share_network:
+        :param std_hidden_sizes: List of sizes for the fully-connected layers
+          for std.
+        :param min_std: Whether to make sure that the std is at least some
+          threshold value, to avoid numerical issues.
+        :param std_hidden_nonlinearity: Nonlinearity used for std network.
+        :param hidden_nonlinearity: Nonlinearity used for each hidden layer.
+        :param output_nonlinearity: Nonlinearity for the output layer.
+        :param mean_network: Custom network for the output mean.
+        :param std_network: Custom network for the output log std.
+        :param std_parametrization: How the std should be parametrized. There
+          are a few options:
+            - exp: The logarithm of the std will be stored, and applied a
+                exponential transformation
+            - softplus: The std will be computed as log(1+exp(x))
+        :return:
+    """
+
+    def __init__(self,
+                 input_dim,
+                 output_dim,
+                 dist=tf.contrib.distributions.MultivariateNormalDiag,
+                 scope="GaussianMLPModel",
+                 hidden_sizes=(32, 32),
+                 learn_std=True,
+                 init_std=1.0,
+                 adaptive_std=False,
+                 std_share_network=False,
+                 std_hidden_sizes=(32, 32),
+                 min_std=1e-6,
+                 max_std=None,
+                 std_hidden_nonlinearity='relu',
+                 hidden_nonlinearity='relu',
+                 output_nonlinearity=None,
+                 std_parameterization='exp',
+                 *args,
```

Comment:
Same here. I can't find where the args and kwargs are used.

Reply:
useless, will remove that.